Doctrine/DataFixtures/AbstractLoader.php

```
@@ -31,6 +31,6 @@
      */
     public function load(ObjectManager $objectManager)
     {
-        return $this->container->get('hautelook_alice.fixtures.loader')->load($objectManager, $this->getFixtures());
+        return [];
```

Might be better to throw an exception instead